client/src/index.tsx

```diff
@@ -25,6 +25,7 @@ root.render(
     <Tooltip.Provider delayDuration={300}>
       <BackgroundColorProvider>
         <RouterProvider router={router} />
+        {/* <RouterProvider router={router} future={{ v7_startTransition: true }} /> */}
```

Comment:
```suggestion
```

Probably something we can do in a future PR :D

Reply:
Yeah good call, removed :D